AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -1331,6 +1331,9 @@ r3owners.net#@#.adsbygoogle
 thesouthern.com,poststar.com,wcfcourier.com##.tnt-ads-container
 business.financialpost.com##.widget_postmedia_layouts_ad
 !#endif
+thesaurus.com#%#AG_abortOnPropertyRead('ENV_VARS');
+!+ PLATFORM(ext_ff, ext_opera, ios, ext_android_cb, ext_safari)
```

Comment:
This rule is required in English/content_blocker too for ext_safari and iOS due to different adblocker modules. Also $domain=thesaurus.com is not required.